patches/components-password_manager-core-browser-login_database_mac.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/password_manager/core/browser/login_database_mac.cc b/components/password_manager/core/browser/login_database_mac.cc
+index 43d3736886ceacf8cb3787382304db404cfc084c..5e1ff136dce58805d441fb7b7ae15d422297b2fe 100644
+--- a/components/password_manager/core/browser/login_database_mac.cc
++++ b/components/password_manager/core/browser/login_database_mac.cc
+@@ -25,7 +25,7 @@ LoginDatabase::EncryptionResult LoginDatabase::DecryptedString(
+   if (OSCrypt::DecryptString16(cipher_text, plain_text))
+     return ENCRYPTION_RESULT_SUCCESS;
+ 
+-  return OSCrypt::IsEncryptionAvailable() && IsUsingCleanupMechanism()
++  return OSCrypt::IsEncryptionAvailable() /* && IsUsingCleanupMechanism() */
```

Comment:
why aren't we using kDeleteCorruptedPasswords to fix this?

Reply:
ok - Using `kDeleteCorruptedPasswords` could work on macOS.

Reply:
@bridiver Fixed.

Reply:
Hmm, I think using `kDeleteCorruptedPasswords` by default seems dangerous.
Brave will clear all passwords if decryption is failed temporarily.
How about using my initial way?